### DISCUSSION TITLE
Proper motions?

### DISCUSSION THREAD

**Discussion by zecevicp** [LSSTDESC/desc-data-portal#83] [Q&A] [answered]
Hi all, 
can you please tell me were the proper motions available in the input catalog and, if they were, is the input catalog still available somewhere? Currently the proper motions don't seem to be in the published DP0 catalogs..

**Comment by rmandelb** [comment, marked as answer]
Proper motion effects were not included in the simulated images.  The reason is outlined in section 4.2 of the [DC2 paper](https://arxiv.org/pdf/2010.05926.pdf) as follows: "Since version 19.0.0 of the LSST Science Pipelines, which we used for image processing, cannot handle proper motion and parallax of stars, we omitted those effects from the instance catalog entries for those objects."  My understanding is that the Galfast catalog that was used as the input for DC2 _did_ have proper motions, but since we did not simulate those proper motions, we did not end up carrying that information around in our catalogs.

**Reply by rmandelb** [reply]
I should clarify: the premise behind my response was that you were asking for the proper motions because you thought they had been included in the image simulations.  If that is incorrect and you had something else in mind, please let me know - I'm happy to rope in others for discussion if needed.

**Comment by zecevicp** [comment]
Yes, that's what I was asking. I was hoping that maybe they were included in the simulations but not published. Thank you for the clarification and the quick response.

**Reply by rmandelb** [reply]
You're welcome.  

For context, the reason for this choice (to not include proper motions in the sim) can be found in the [paper describing an earlier simulation campaign called DC1](https://arxiv.org/pdf/2001.00941.pdf), section 5.1 and figure 5.  We found that with versions of the LSST Science Pipelines that were not set up to handle proper motions, this uncorrected effect totally dominated the astrometric residuals.  The effect was therefore omitted so that it would not dominate the tests of the astrometry, and we'd be able to pick up other issues that affect the astrometry.
